pandas/tests/io/formats/test_style.py

```diff
@@ -102,12 +102,21 @@ def test_deepcopy(self):
         assert self.styler._todo != s2._todo
 
     def test_clear(self):
-        s = self.df.style.highlight_max()._compute()
-        assert len(s.ctx) > 0
+        tt = DataFrame({"A": [None, "tt"]})
+        css = DataFrame({"A": [None, "cls-a"]})
+        s = self.df.style.highlight_max().set_tooltips(tt).set_td_classes(css)
         assert len(s._todo) > 0
+        assert s.tooltips
+        assert len(s.cell_context) > 0
+        s = s._compute()
```

Comment:
is there a reason we are calling private methods?

Reply:
this is legacy from my perspective. `_compute` is wrapped up within the render process. Calling just this, rather then say `render` is just an optimisation of the test, I believe.

Reply:
ok, yeah i fyou can just add some blank lines here so its a bit more obvious what is tested